pkg/database/postgres/tables.go

```diff
@@ -213,9 +213,9 @@ order by c.ordinal_position`
 }
 
 func (p *PostgresConnection) GetTableSchema(tableName string) ([]*types.Column, error) {
-	query := "select column_name, data_type, character_maximum_length, column_default, is_nullable from information_schema.columns where table_name = $1 and table_schema = $2"
+	query := "select column_name, data_type, character_maximum_length, column_default, is_nullable from information_schema.columns where table_name = $1 and catalog = $2"
 
-	rows, err := p.conn.Query(context.Background(), query, tableName, p.databaseName)
+	rows, err := p.conn.Query(context.Background(), query, tableName, "public")
```

Comment:
Switch this back to p.databaseName to match table_catalog

I should have mentioned it with the previous comment.

Reply:
:ok_hand: